client/src/actions/index.js

```diff
@@ -243,12 +242,16 @@ const requestDifferentialExpression = (set1, set2, num_genes = 10) => async (
     const state = getState();
     const { universe } = state.controls;
 
+    // Legal values are null, Array or TypedArray.  Null is initial state.
+    if (!set1) set1 = [];
```

Comment:
did this change have to do with the dataframes or just general improvements?

Reply:
latent bug found by visual inspection.   Given that the code in this general area is changing, it made sense to fix it.